### PR TITLE
Temp dev

### PR DESCRIPTION
**Pull request by agrawalshanu708**

### CODE REVIEW

**Review by tushargoyal812 — COMMENTED** [review]
file structure is good.
variable naming is good.
variable naming is good.

File: src/App.css [hunk 1]
```diff
@@ -2,11 +2,25 @@ body {
...
  --light-black: #18181b;
  --white-color: #ffffff;
  --primary-color: #AB542F;
-}
\ No newline at end of file
+}
+.btn{
+  color: var(--white-color);
+  background-color: var( --primary-color) ;
+  border: 0;
```
Comment (MonikaJhingan) [inline src/App.css:17]: Use of var is done which is good.

**Review by MonikaJhingan — APPROVED** [review]
The code looks well indented as well as implemented.

**Review by sashaboi — APPROVED** [review]
Code is very readable and review friendly, 
Liked the folder structure , it clearly shows how the data is flowing.
Looks good to merge!

**Review by anshtripathi01 — APPROVED** [review]
Code Looks good to merge

**Pull request merged**